cellmlmanip/model.py

```diff
@@ -362,61 +378,135 @@ class QuantityStore(object):
     """
 
     # Aliases for units to their Sympy equivalent
-    aliases = {
+    UNIT_ALIASES = {
         'litre': 'liter',
-        'litres': 'liters',
         'metre': 'meter',
-        'metres': 'meters'
     }
 
+    # The full list of supported CellML units
+    # Taken from https://www.cellml.org/specifications/cellml_1.1/#sec_units
+    # Some are not defined by Sympy, see comments
+    CELLML_UNITS = [
+        # Base SI units
+        'ampere',
+        'candela',
+        'kelvin',
+        'kilogram',
+        'meter',
+        'metre',  # in self.UNIT_ALIASES
+        'mole',
+        'second',
+
+        # Derived SI units
+        'becquerel',  # see __add_custom_units()
+        'celsius',  # TODO: not defined by Sympy
```

Comment:
I'd instead add a comment saying this is removed in CellML 2 and won't be supported by us :)

Reply:
Gladly!